File: cpp/919B.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;
using ll = long long;


ll digitSum(ll number){
    ll result = 0;

    while (number != 0){
        result += number % 10;
        number /= 10;
    }

    return result;
}


int main(){

    ll k;
    cin >> k;

    ll number = 0;
    while(k != 0){
        number++;
        if (digitSum(number) == 10){
            k--;
        }
    }

    cout << number;

    return 0;
}

File: python/919B.py


def digit_sum(number):
    result = 0
    while number > 0:
        result += number % 10
        number //= 10
    return result


k = int(input())
number = 0

while k != 0:
    number += 1
    if digit_sum(number) == 10:
        k -= 1

print(number)
